sdk/servicebus/azure-servicebus/tests/servicebus_preparer.py

```diff
@@ -22,13 +26,116 @@
 SERVICEBUS_QUEUE_PARAM = 'servicebus_queue'
 SERVICEBUS_AUTHORIZATION_RULE_PARAM = 'servicebus_authorization_rule'
 SERVICEBUS_QUEUE_AUTHORIZATION_RULE_PARAM = 'servicebus_queue_authorization_rule'
+SERVICEBUS_ENDPOINT_SUFFIX = os.environ.get('SERVICEBUS_ENDPOINT_SUFFIX', 'servicebus.windows.net')
```

Comment:
In the tests below the suffix was `.servicebus.windows.net`, so it might need the same here